excellent/evaluator_test.go

```diff
@@ -341,10 +340,9 @@ var errorTests = []struct {
 	errorMsg string
 }{
 	// parser errors
-	{`@(")`, `error evaluating @("): syntax error at "`},
```

Comment:
This is no longer an expression error because it's no longer an expression. The `"` starts a string literal that never closes so the expression never closes.